auth_brute_force/controllers/main.py

```diff
@@ -45,13 +47,13 @@ def web_login(self, redirect=None, **kw):
                         request.session.db, request.params['login'],
                         request.params['password'])
                 # Log attempt
-                attempt_obj.create({
+                attempt_id = attempt_obj.create({
```

Comment:
minor changes, but the name is better attempt, than attempt_id, because it is an object.

Reply:
Done